nix/default.nix

```diff
@@ -134,7 +111,8 @@ let
           install-spago-style
         '';
       buildPhase = ''
-        build-spago-style "./**/*.purs"
+        psa --strict --censor-lib --is-lib=.spago ${spagoGlobs} \
+          --censor-codes=UserDefinedWarning "./**/*.purs"
```

Comment:
This is great! I wish psa had config file to manage reporting like hlint does. Consider adding `censorCodes` argument with `[ "UserDefinedWarning" ]` as default